mne/io/base.py

```diff
@@ -1740,9 +1774,6 @@ def append(self, raws, preload=None):
             else:
                 preload = False
 
-        if not preload and not isinstance(self, (Raw, RawKIT, RawEDF)):
-            raise RuntimeError('preload must be True to concatenate '
-                               'files unless they are FIF, KIT, or EDF')
```

Comment:
I suppose it is okay to remove this? I don't know why this was here in the first place.